.gitignore

```diff
@@ -12,3 +12,6 @@
 *.log
 *.swp
 *~
+:memory
+:memory-shm
+:memory-wal
```

Comment:
`Ronin::DB.connect('sqlite3::memory')` cause these files to appear to I added them to `.gitignore`.

Reply:
Er wait, my mistake, it should be `sqlite3::memory:`. That will prevent those files from being created.

Reply:
😅